dashboard/test/models/user_test.rb

```diff
@@ -4768,6 +4769,19 @@ def hide_scripts_in_sections(section1, section2)
     assert_equal 1, user.properties['section_attempts']
   end
 
+  test 'family name is added to summarize' do
+    user = create :user
+    family_name = 'TestFamilyName'
+    user.properties = {family_name: family_name}
+
+    DCDO.stubs(:get).with('family-name-features', false).returns(true)
```

Comment:
I'd like to see this test also cover the default case, with something like `assert_nil(user.summarize[:family_name])`

Reply:
do you mean the case where the flag is false? I can do that